src/gsi/read_aerosol.f90

```diff
@@ -421,12 +434,20 @@ subroutine read_aerosol(nread,ndata,nodata,jsatid,infile,gstime,lunout, &
               endif
 
               !    extract header information
-              call ufbint(lunin,hdrvaodg,13,1,iret,vaodgstr)
+              if (.not. new_viirsTab) then
+                call ufbint(lunin,hdrvaodg,13,1,iret,vaodgstr)
+              else
+                call ufbint(lunin,hdrvaodg,13,1,iret,vaodgstr1)
+               ! write(*,'(a20,8f10.2,3f20.5,2f7.2)') "read_aero_hdrvaodg ",hdrvaodg 
+              endif
               rsat = hdrvaodg(1); ksatid=rsat
 
-              if ( jsatid == 'NPP' .or. jsatid == 'npp' ) kidsat = 225
+              !if ( jsatid == 'NPP' .or. jsatid == 'npp' ) kidsat = 225
```

Comment:
Remove this if it is the wrong code

Reply:
removed.

Reply:
The correct sat id for NPP should be 224.